dbt/models/model/schema.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+
+
+models:
+  - name: assessment_card
+  - name: assessment_pin
+  - name: feature_importance
+  - name: final_model
+  - name: metadata
+  - name: parameter_final
+  - name: parameter_range
+  - name: parameter_search
+  - name: performance
+  - name: performance_quantile
+  - name: shap
+  - name: test_card
```

Comment:
Do we actually want to keep `test_card` in the DAG, or was this a temporary testing table?

Reply:
Gotcha, I'll leave this in then!